pkg/docker/util.go

```diff
@@ -16,8 +16,8 @@ import (
 	"github.com/openshift/source-to-image/pkg/util/user"
 )
 
-// DockerImageReference points to a Docker image.
-type DockerImageReference struct {
+// ImageReference points to a Docker image.
+type ImageReference struct {
```

Comment:
Most of this PR is nice cleanup, but changes like this one strike me as arbitrary and unnecessary code churn.  what's the motivation to rename this?